specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,36 @@
+### SKU Price ID
+
+A SKU Price ID is a unique identifier that defines the unit price associated with the SKU of the charge. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU. The composition of the detailed information associated with the price may differ from one provider to the next.
```

Comment:
```suggestion
A SKU Price ID is a unique identifier that defines a unit price associated with the SKU of the charge. If providers define their prices at the SKU level, the SKU ID and and SKU Price ID shall be the same value. Every SKU must have one or more SKU prices, identified by SKU Price ID. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the identified unit price associated with the SKU. The composition of the detailed information associated with the price may differ from between providers.
```

Consolidating description && being more explicit about nature of relationship of skuPriceId : skuId

Reply:
@udam-f2 just finessed this section earlier today, so I'd want his input.

Reply:
@aqu-erp why 'a unit price' vs 'the unit price'? The sku price id is the unique ID that we want to see here that's associated with the charge right?

Reply:
To my ear, 'the unit price' implies that the identified price is the only price for the SKU, which may not be true

> that defines `the only` unit price associated with the SKU of the charge

whereas, 'a unit price' leaves room for one or many

> that defines `a` unit price `of potentially many that are` associated with the SKU of the charge

Reply:
Per discussion at the member meeting, SKU ID will now be defined by SKU Price ID, and therefore much of this language will be moved to the definition for SKU ID.